_data/domains.yml

```diff
@@ -438,6 +438,13 @@ websites:
       - software
     doc: https://www.pairdomains.com/kb/posts/345
 
+  - name: Park.io
+    url: https://park.io/
+    img: parkio.png
+    tfa:
+      - software
```

Comment:
Please don't use the software tag. Use proprietary or totp depending what the site supports.
See [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#new-sites)

Reply:
Park.io supports Authy accounts (7-digit codes). It won't work with, say, Google Authenticator.

Do you consider this proprietary or standard TOTP?

Reply:
That's a proprietary system (aka non RFC-6238).